Failed to build for Ubuntu 14.04 (PPA)

Hello. I tried to build Lumina 0.8.8 for Ubuntu 14.04 in PPA. But build failed with error:

```
make[2]: Leaving directory `/«PKGBUILDDIR»/lumina-fileinfo'
make[1]: Leaving directory `/«PKGBUILDDIR»'
   debian/rules override_dh_install
make[1]: Entering directory `/«PKGBUILDDIR»'
dh_install --list-missing
dh_install: libluminautils1 missing files (usr/lib/*/libLuminaUtils.so.1), aborting
make[1]: *** [override_dh_install] Error 20
make[1]: Leaving directory `/«PKGBUILDDIR»'
make: *** [binary-arch] Error 2
dpkg-buildpackage: error: fakeroot debian/rules binary-arch gave error exit status 2
```

Here is a full buildlog http://pastebin.com/K6LHkUSr

--- a/debian/libluminautils-dev.install
+++ b/debian/libluminautils-dev.install
@@ -1,2 +1,2 @@
-usr/lib/*/libLuminaUtils.so
+usr/lib/libLuminaUtils.so
 usr/include/*.h

--- a/debian/libluminautils1.install
+++ b/debian/libluminautils1.install
@@ -1,3 +1,3 @@
-usr/lib/*/libLuminaUtils.so.1
-usr/lib/*/libLuminaUtils.so.1.0
-usr/lib/*/libLuminaUtils.so.1.0.0
+usr/lib/libLuminaUtils.so.1
+usr/lib/libLuminaUtils.so.1.0
+usr/lib/libLuminaUtils.so.1.0.0

